firewood/src/manager.rs

```diff
@@ -76,7 +76,7 @@ pub(crate) struct RevisionManager {
     /// stored in the filebacked storage.
     historical: RwLock<VecDeque<CommittedRevision>>,
     proposals: Mutex<Vec<ProposedRevision>>,
-    // committing_proposals: VecDeque<Arc<ProposedImmutable>>,
+    /// The list of revisions that are on disk by hash.
```

Comment:
This comment will become incorrect when deferred persistence is implemented. I think it's self-documenting already so the doc comment doesn't add much value.

The `committing_proposals` thing is dead code that should probably go away.